hydroflow_plus_test/src/lib.rs

```diff
@@ -6,9 +6,24 @@ use hydroflow_plus::tokio_stream::wrappers::UnboundedReceiverStream;
 use hydroflow_plus::*;
 use stageleft::{q, Quoted, RuntimeData};
 
+#[cfg(feature = "macro")]
+pub(crate) mod cluster;
+#[cfg(not(feature = "macro"))]
 pub mod cluster;
```

Comment:
These seem backwards? features need to be additive?

Reply:
This is related to above hack, we want to not have `pub mod` when we are compiling the top level `lib` for a macro crate.